Makefile

```diff
@@ -79,4 +79,10 @@ py-pb:
 		.;
 	cp -a output/bess_pb/. ${PTF_PB_DIR}
 
-.PHONY: docker-build docker-push output pb
+fmt:
+	@gofmt -s -l -w $(GO_FILES)
+
+golint:
+	@docker run --rm -v $(CURDIR):/app -w /app/pfcpiface golangci/golangci-lint:latest golangci-lint run -v --config /app/.golangci.yml
+
+.PHONY: docker-build docker-push output pb fmt golint
```

Comment:
@dariusgrassi should this change be here? We might have resolved merge conflicts in the wrong way

Reply:
Nevermind... I was looking at one commit only, not the full PR diff. Please disregard.